homework-pair.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+function is_pair(string $sequence): bool
```

Comment:
- The `is_anagram3()` must return a boolean, it returns a string now
- The function doesn't respond correctly, I tried it with the `((()))` input